boards/feather-m0/Makefile.features

```diff
@@ -0,0 +1,14 @@
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_adc
+FEATURES_PROVIDED += periph_cpuid
```

Comment:
Is now already in sam0_common/Makefile.features (see #7880)

Reply:
Fixed !